Changing `math` module calls into `ansys-math-core` ones (#2102)
ansys/pymapdl: doc/source/user_guide/math.rst

.. _mapdl_math_class_ref:

PyAnsys Math overview
=====================
`PyAnsys Math <pyansys_math_>`_ provides the ability to access and manipulate

> the large sparse matrices and solve a variety of eigenproblems in a similar

Reviewer: ```suggestion
large sparse matrices and solve a variety of eigenproblems in a similar
```

manner to the popular `numpy <numpy_docs_>`_ and `scipy <scipy_docs_>`_ libraries.

PyMAPDL and PyAnsys Math
~~~~~~~~~~~~~~~~~~~~~~~~
This example demonstrates how to take advantage of the `ansys-math-core` package
with PyMAPDL.

.. This example demonstrates how to send an MAPDL Math matrix from MAPDL
.. to Python and then send it back to be solved. While this example runs the
.. :func:`mm.eigs() <ansys.math.core.math.AnsMath.eigs>` method on mass
.. and stiffness matrices generated from MAPDL, you could instead use
.. mass and stiffness matrices generated from an external FEM tool or
.. even modify the mass and stiffness matrices within Python.

.. First, solve the first 10 modes of a ``1 x 1 x 1`` steel meter cube
.. in MAPDL.

> 

Reviewer 2: Is this intended to be a comment?

Reviewer 3: Looks like it is just coming from the old description, please uncomment and adapt, or remove. As you prefer.

    (241, 241)	50854700854.68495
    (242, 242)	95726495726.47179

> Another way to call PyAnsys Math directly from PyMAPDL exists
> with the following command:

Reviewer: ```suggestion
You can also call PyAnsys Math directly from PyMAPDL with this command:
```
